deployments/container/nvidia-mig-manager-example-hopper-blackwell.yaml

```diff
@@ -276,11 +276,17 @@ data:
             "7g.94gb": 1
     
       # GB200 HGX
-      all-1g.24gb.me:
+      all-1g.23gb:
+          devices: all
+          mig-enabled: true
+          mig-devices:
+            "1g.23gb": 7
+
+      all-1g.23gb.me:
```

Comment:
Instead of updating ` all-1g.24gb.me:` to ` all-1g.23gb.me:`, let's add `all-1g.24gb.me:` to this file and leave the former as-is